index.js

```diff
@@ -182,6 +220,8 @@ app.post("/", (req, res) => {
 app.listen(8545);
 
 setInterval(() => {
+  reorganizeWithProbabilityAndTimeout(0.33,1000);
+  reorganizeWithProbabilityAndTimeout(0.33,1000);
```

Comment:
```suggestion
  reorganizeWithProbabilityAndTimeout(0.33,1500);
```

Otherwise it will trigger at the same time when both dices hit :)